Packs/Cisco-umbrella/Integrations/Cisco-umbrella-investigate/Cisco-umbrella-investigate.py

```diff
@@ -871,16 +886,20 @@ def get_domain_command():
                                                      'Message': 'No results found',
                                                      'Reliability': reliability}
 
-                results.append({
-                    'Type': entryTypes['note'],
-                    'ContentsFormat': formats['json'],
-                    'Contents': contents,
-                    'HumanReadable': human_readable,
-                    'HumanReadableFormat': formats['markdown'],
-                    'EntryContext': context
-                })
+                results.append(
+                    CommandResults(
+                        readable_output=human_readable,
+                        outputs=context,
+                        raw_response=contents
+                    ))
             else:
-                raise r
+                if execution_metrics.metrics is not None and execution_metrics.is_supported():
+                    results.append(execution_metrics.metrics)
```

Comment:
This logic gets used quite a bit. It may be worth seeing what we can do to move this into CommonServerPython. Maybe something like:

```python
def append_metrics(execution_metrics, results):
   if execution_metrics.metrics is not None and execution_metrics.is_supported():
                    results.append(execution_metrics.metrics)
   return results
```

Then just call `results = append_metrics(execution_metrics, results)` before returning the results.

Just a thought.